Makefile

```diff
@@ -339,7 +407,7 @@ verify-config-commited: _verify-config
 	fi; \
 
 .PHONY: gqlgen
-gqlgen:
+gqlgen: gen-protobuf
```

Comment:
Why gqlgen depends on gen-protobuf?  It does not affect generated code of gqlgen in any way (which is already committed). 

What target really depends on gen-protobuf?

Reply:
I have removed dependencies to gen-protobuf. I'm following the same patter we used for GQL code generation